tests/integration/tests.mk

```diff
@@ -55,6 +55,29 @@ test.integration.%.kube: | $(JUNIT_REPORT)
 	${_INTEGRATION_TEST_INGRESS_FLAG} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
 
+# Test targets to run with the new installer. Some targets are filtered now as they are not yet working
```

Comment:
Can you elaborate a little bit more why clean up and set up are disabled?

Reply:
We install istio first then run the tests against the existing install. If we don't disable cleanup/setup it will try to install istio during the tests (with old install)

Reply:
Ok, I understand setup, why disable cleanup? what is consequences of not cleaning up?

Reply:
cleanup would delete the install we have **during** the test, because the cleanup runs multiple times during the test not just at the end. 

Prow will tear down the cluster after the test is done so the test doesn't need to do the cleanup itself. You can't trust the tests to ever clean up, because if they crash, etc they wouldn't.

Reply:
Actually now that I think about it, maybe cleanup will only cleanup things it creates and not mess with the existing install. I will test it out.

Reply:
Tested it out. It works with or without cleanup. I don't have a strong preference either way

Reply:
Let's leave as is, no clean up no setup.